### DISCUSSION TITLE
Search field for Shortcuts

### DISCUSSION THREAD

**Discussion by hsandt** [Orama-Interactive/Pixelorama#983] [Ideas]
I wanted to check if there was any shortcut for Move Layer Up/Down, so I opened Preferences > Shortcuts, but there is a lot of group and sub-groups to unfold to search for a specific command and I have no idea where it would be since Move Layer Up/Down is not a top-menu command but a Layer panel button (I also expected a top-menu item Layer > Move Up/Down to be fair, it would help to bind the associated shortcut).

Being able to search through shortcuts by name (and also by binding to avoid conflicts) would help.

**Comment by Variable-ind** [comment]
These shortcuts don't exist yet, but they're planned and may get added at some point
